user-guide/Standard_Apps/EPM/EPM_D-DOCSIS/D-DOCSIS_parameters/D-DOCSIS_parameters_rpd.md

```diff
@@ -301,6 +301,202 @@ This page contains an overview of the RPD parameters available in the D-DOCSIS b
 
   This value is obtained by counting unique Core MAC associations from the RPD CCAP Cores table (MIB OID: 1.3.6.1.4.1.4491.2.1.30.1.1.4.1.12) and the RPD RPM Cores table (Vecima RPM Apigee: rpd/mac-address;rpd/ccap-cores/ccap-core/state).
 
+- **Number Cores**: Calculated. The number of cores, including remote, associated with the RPD.
```

Comment:
This is a duplicate of the Number Cores parameter above it. Is this a mistake, or should this parameter indeed be mentioned twice?

Reply:
Ah, yes, this was a mistake on my part and can be deleted. Thank you